pyproject.toml

```diff
@@ -83,5 +83,38 @@ artifacts = ["/backend/fastrtc/templates", "*.pyi"]
 packages = ["/backend/fastrtc"]
 
 [tool.ruff]
+src = ["demo", "backend/fastrtc"]
 target-version = "py310"
 extend-exclude = ["demo/phonic_chat", "demo/nextjs_voice_chat"]
+
+[tool.ruff.format]
+exclude = ["*.pyi"]
+quote-style = "double"
+indent-style = "space"
+skip-magic-trailing-comma = false
+line-ending = "auto"
+
+[tool.ruff.lint]
+select = [
+  "E",
+  "F",
+  "W",
+  "Q",
+  "I",
+]
+
+# These can be turned on when the framework is more mature (Too many errors right now)
+exclude = [
+  "D",
+  "UP"
```

Comment:
Supporting the modern syntax could be nice eventually

Reply:
I was thinking of introducing this in a follow-up PR right after this one. It is roughly 20-30 type "violations" changing Dict->dict, etc.

Reply:
Yea that would be great and welcome!